### PR TITLE
Help update

### PR DESCRIPTION
**Pull request by areeda**
this PR just updates the doc strings of each gwpy-plot product so that the help message is more informative.

### CODE REVIEW

File: gwpy/tests/test_io.py [hunk 1]
```diff
@@ -35,8 +35,12 @@
                      gwf as io_gwf,
                      kerberos as io_kerberos,
                      nds2 as io_nds2,
+<<<<<<< HEAD
```
Comment (duncanmmacleod) [inline gwpy/tests/test_io.py]: @areeda, please resolve this conflict, this is the primary reason for the travis-ci failures.

**Comment by duncanmmacleod** [comment]
@areeda, how are you updating your fork relative to the upstream? having ~100 commits from me in an otherwise tiny PR is very confusing.

**Comment by duncanmmacleod** [comment]
@areeda, is there an update on this PR? Are you able to remove the conflict commit?

**Comment by duncanmmacleod** [comment]
@areeda, I'm not happy with 75cf2aedaf1565d29be63a306401618c016aecdc. It seems to replace some symlinks with copies of files, and removes a normal carriage return character from `gwpy.io.nds2` for no apparent reason. Can you drop this commit (via `git rebase -i HEAD~3`)?

**Comment by areeda** [comment]
@duncanmmacleod I don't understand why but rebase always give me incomprehensible errors.
```$git rebase -i HEAD~3
Auto-merging bin/gwpy-plot
CONFLICT (content): Merge conflict in bin/gwpy-plot
error: could not apply 2f8304ea... bin/gwpy-plot: add result as a local variable containing data used for

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".

Could not apply 2f8304ea... bin/gwpy-plot: add result as a local variable containing data used for graph. This is useful in interactive mode.```

**Comment by areeda** [comment]
then when I resolve one multiple others come up
```areeda@sylvester:gwpy[610]$git checkout --ours bin/gwpy-plot
areeda@sylvester:gwpy[611]$git add bin/gwpy-plot
areeda@sylvester:gwpy[612]$git commit
interactive rebase in progress; onto 76539aa5
Last command done (1 command done):
   pick 2f8304ea bin/gwpy-plot: add result as a local variable containing data used for graph. This is useful in interactive mode.
Next commands to do (203 remaining commands):
   pick 123c26c8 gwpy/cli/cliproduct.py: add result to member variables
   pick 27e1c9ce gwpy/cli/spectrogram.py: store results in object for interactive mode
You are currently rebasing branch 'help-update' on '76539aa5'.

nothing to commit, working tree clean
areeda@sylvester:gwpy[613]$git rebase --continue
Auto-merging gwpy/cli/cliproduct.py
CONFLICT (content): Merge conflict in gwpy/cli/cliproduct.py
error: could not apply 123c26c8... gwpy/cli/cliproduct.py: add result to member variables

When you have resolved this problem, run "git rebase --continue".
If you prefer to skip this patch, run "git rebase --skip" instead.
To check out the original branch and stop rebasing, run "git rebase --abort".

Could not apply 123c26c8... gwpy/cli/cliproduct.py: add result to member variables```

**Comment by duncanmmacleod** [comment]
@areeda, I'm not sure if there's an easy way to fix the problems here. I recommend trying to set up a new branch (up-to-date relative to `upstream/develop`) and include _only_ the relevant commits for these changes.

**Comment by duncanmmacleod** [comment]
This has been replaced by #626 (pending approval from @areeda).

**Pull request closed without merging**